src/components/inputs/SuggestSelect.jsx

```diff
@@ -387,6 +387,7 @@ const SuggestSelect = React.createClass({
 
   onOptionSelect(option, e) {
     this.selectOption(option.id);
+    if (!this.isMounted()) e.stopPropagation();
```

Comment:
a bit worried about that
https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html